### DISCUSSION TITLE
Clips only looping first bar when triggering scene

### DISCUSSION THREAD

**Discussion by ll-dasein-ll** [KBLiveSolutions/open.control#25] [Troubleshooting]
I've noticed a strange problem regarding triggering scenes - if I use the OpenControl to fire off a scene, only the first bar of the clips will play and loop continuously, rather than looping the expected length of the clips in the scene.  If I use OpenControl to trigger the clip, it will play and loop the full clip as normal.  There are no settings/follow actions or anything set up on the clips or scenes which could explain the behaviour.  If I use the scene buttons on my APK keyboard to immediately fire off the same scenes they will play as expected, looping the full length of clips.  They also play normally if I click the play button on the scenes with the cursor.

I've noticed that this same behaviour happens if I use my monome 40H and the Terms-Launchd device to trigger scenes so it appears that it's not specifically an OpenControl issue, but I can't find any solution or explanation online.  Has anyone got any suggestions to explain what might be causing this?  It's really frustrating!

**Comment by KBLiveSolutions** [comment]
Hello
I'm sorry but I can't reproduce this.
Can you try to deactivate everything but open·control, and see if this is still happening ?

**Comment by aladinsound** [comment]
I also have this probleme..
Since I received my open.control last week, I used it a lot everyday. For me it's a really game changer!
But Today it suddenly begun to do this problem :

when I launch a scene it plays the clips only the duration of the quantized value. (for example my clip last 16 bars and is set to quantize launch at 1 bar > it will only play one bar, if i change the clip to set the launch quantize at 4 bars it will plays 4 bars!)
I tried to open a completly new projetc, the probleme is the same, I tried to remove all my others controllers and ports in the midi preferences of live. The problem still remain...
also this problem doesn't happen with my Push2.
any idea?
I have a show tomorow, I was so happy to use the controller...
i'd be sad if it didn't work....
cheers!

**Reply by KBLiveSolutions** [reply]
Hi

So, you have a button on open·control set to "Launch Scene", and when you press it, the clip plays for the length of the global quantization, and then stops ? I have tried, but I can't manage to reproduce that.

Can you please send a simple set that reproduces it, and also export the open·control page setting (on the Controller Editor, click on the Export button below the Page select menu) ?

**Reply by ll-dasein-ll** [reply]
I completely forgot about this thread - I've got a new M1 Macbook Pro since I last posted and the same thing still happens on it too.  It only seems to happen if I've got another MIDI controller connected to Live too - like my Launchpad.  When the open control is the only controller connected it doesn't seem to happen. I've not had much chance to experiment for a few weeks though - I'll try again on Friday and assuming I can reproduce it I'll share a set that has the issue.  The set that I'm working on at the moment is huge, so I'll delete a load of channels and scenes from it and send a compact version.

**Reply by aladinsound** [reply]
thank you for your answer!

here is the config of my controller:

<img width="1437" alt="Capture d’écran 2022-09-06 à 22 14 28" src="https://user-images.githubusercontent.com/112964122/188729883-897fa403-a6b1-4f7c-a1d5-51e4cebebb37.png">

<img width="1438" alt="Capture d’écran 2022-09-06 à 22 14 50" src="https://user-images.githubusercontent.com/112964122/188729933-b8150666-c4ca-466c-9de6-ef14a500c65d.png">

regarding ableton, for me it happens also on a new project created from scratch.
and it happens also without any other controller plugged.

" So, you have a button on open·control set to "Launch Scene", and when you press it, the clip plays for the length of the global quantization, and then stops "
No, it plays it as a loop (if loop is selected) with the length of the quantize value.

**Reply by KBLiveSolutions** [reply]
if you have several clips on different tracks, they all loop with the length of the global quantization?
do the clips play button blink? this would mean that they are re-launched, or that the scene gets relaunched.

I still can't reproduce it with the same open·control setting.
if you have a simple set that reproduces it, that would be nice.

**Reply by aladinsound** [reply]
> if you have several clips on different tracks, they all loop with the length of the global quantization?

they loop to the length of their quantization. If I set different quantizations values on differents clips, every clip will loop at the value of its quantization.
It happens only when I launch with Open.control.

> do the clips play button blink? this would mean that they are re-launched, or that the scene gets relaunched. 

I have to check tomorrow because I don't have the open control with me tonight.

> I still can't reproduce it with the same open·control setting.
if you have a simple set that reproduces it, that would be nice.

Yes, no problem. I will get the file tomorrow and send it to you.

**Reply by aladinsound** [reply]
> do the clips play button blink? this would mean that they are re-launched, or that the scene gets relaunched.

no, the clips play button doesn't blink.

here is a simple project with this problem :

[test lancement Open Control Project.zip](https://github.com/KBLiveSolutions/open.control/files/9505120/test.lancement.Open.Control.Project.zip)

thank you!

**Reply by KBLiveSolutions** [reply]
Ok, I see where the problem comes from : it's because you have set your clips on **Launch : Repeat**.
I'm not sure how to fix that (and quickly enough for your gig). But if it's possible for you, you can set them to **Launch : Trigger**, and the clips will behave correctly.
Let me know if that's ok for you until I fix that.

Cheers

**Reply by aladinsound** [reply]
Thank you a lot for your answer.

I will do my show with an other controller as usual to launch scenes. I will use the open.control for the other fonctions (especially the name display wich is a huge feature for my workflow!..) until you can fix it.

**Reply by KBLiveSolutions** [reply]
ok, I'll see what I can do.
Just out of curiosity : is there a specific reason your clips are set to Repeat ?

**Reply by ll-dasein-ll** [reply]
I think you might be into something with the Launch mode setting - I've no idea why because my Prefs are set to use Trigger as default, but I've just loaded the set I'm working on (and which had this problem) and loads of the clips are set to Repeat.  I've just set everything in the set to Trigger instead and I'm going to have a playthrough shortly and see if it's resolved it.  I'm very hopeful!  Thank you in advance - I'll post again when I can confirm it's working a bit later 😁

**Reply by ll-dasein-ll** [reply]
Can confirm the Launch mode set to Repeat was exactly the problem.  All working perfectly now - thank you so much, I'd never have figured that out myself!  This thing is awesome in conjunction with a Launchpad and an APC Key 25.  So happy I took a chance on it when I saw you had more up for sale 😁

**Reply by aladinsound** [reply]
Yeah! It was the same for me, I never asked myself about the launching mode because it was working.
anyway now it's working very good in trigger mode now!
thank you a lot!
